app/views/discovery_rules/_form.html.erb

```diff
@@ -6,7 +6,27 @@
                      :control_group_id => 'search_group', :size => 'col-md-9' %>
   <%= select_f f, :hostgroup_id, accessible_hostgroups, :id, :to_label, { :include_blank => false },
     { :help_inline => _('Target host group for this rule with all properties set')} %>
-  <%= text_f f, :hostname, :help_inline => _('Target hostname template pattern (empty for MAC)')  %>
+  <%= text_f f, :hostname, :help_inline => popover(
+    _("Template"),
```

Comment:
I can't think of anything better other then changing this from inline help to just a popover under the hostname which can give it a little more room.
I'll merge as is.